docs/contributing.md

```diff
@@ -41,7 +41,7 @@ Because this project exports a library that will be used by other projects, it i
 - [Prettier](https://prettier.io/), [TypeScript compilation](https://www.typescriptlang.org/), [eslint](https://eslint.org/) and [stylelint](https://stylelint.io/) are run on _staged files_ as a pre-commit hook
   - For an optimal developer experience, it's recommended that you configure your editor to run linting & formatting inline.
   - These checks will also be run on all files in CI, and must pass before the branch can be merged
-- [`commitlint`](https://github.com/conventional-changelog/commitlint) is run on your commit message, using the [conventional commits config](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional)
+- All pull requests opened into `develop` or `master` will have their title linted according to the [conventional commits spec](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional). The PR title must represent a conventional commit message in order to generate an automated changelog entry. The project is configured to only allow [squash & merge](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) PR commits.
```

Comment:
Potential reword: "All pull requests opened into `develop` or `master`must have a title that follows the [conventional commits spec](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional). This generates an automated changelog entry and is required to merge.  The [WIP] prefix can also used indicate that a pull request is still work in progress. In this case the PR title isn't validated and the pull request linting check remains pending." 

Tried to add in the thing about WIP and also make it clearer that you must have this type of title to merge.

Reply:
@haworku I like it! do you want to insert as a suggestion in the PR?

Reply:
```suggestion
- All pull requests opened into `develop` or `master` must have a title that follows the [conventional commits spec](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional). This generates an automated changelog entry and is required to merge. The [WIP] prefix can also used indicate a pull request is still work in progress. In this case,  the PR title is not validated and the pull request lint check remains pending. 
- The project is configured to only allow [squash & merge](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) PR commits.
```